### ISSUE TITLE
Machine-readable way to define a namespace prefix

### ISSUE THREAD

**Issue by merkys** [Materials-Consortia/definition-provider-template#3]
Currently there is no machine-readable way to define the namespace prefix used in the namespace repository. It might not be needed at all, and possibly can be given in some external link aggregator, however, I can think of at least one use: validation of property names (they should start with a common suffix).

**Comment by ml-evs** [comment]
My understanding was that it is generated by the folder hierarchy, so that it does not need to be explicitly listed in each property definition itself (which could be error prone) (cf. the other issue on the stability repo)

**Comment by merkys** [comment]
Good point, but I do not see where from the prefix could be taken. One possibility is to look at the (supposedly) single file under `src/$VERSION/standards/`, but it would be nice to have a clarification about this.
